README.md

```diff
@@ -5,27 +5,117 @@
 [![Gem Version](https://badge.fury.io/rb/wabur.svg)](https://rubygems.org/gems/wabur)
 [![Gem](https://img.shields.io/gem/dt/wabur.svg)](https://rubygems.org/gems/wabur)
 
-WABuR is a Web Application Builder using Ruby. It employs a modern NoSQL JSON
-data store and a single-page UI using JavaScript. The best part is that it is
-simple and very fast, hitting over 200,000 fetches a second with a Ruby core!
+WABuR is a Web Application Builder using Ruby. It is easy to use, taking just
+a minute to create a hello world web application and it is _FAST_, hitting
+over 200,000 fetches a second with a Ruby core! It employs a modern NoSQL JSON
+data store and a single-page UI using JavaScript.
 
 It is pluggable and extendable in many ways to allow new additions,
 alternative databases, and any number of UIs.
 
-A natural question is *"What about Rails?"*. Rails is well established and has
-a huge user base. WABuR is not a replacement for Rails. It is an alternative
-for those who want to explore using JSON databases with a single-page dynamic
-JavaScript UI.
+## Simple to use Quick Start
+
+With WABuR you are up and running in minutes with only one file to modify to
+specify attributes. It doesn't get any simplier. Heres how.
+
+Install the wabur gem.
+
+```
+$ gem install wabur
+```
+
+Create a new project and cd into the directory.
 
-For further reading there is an [architecture page](pages/Architecture.md)
+```
+$ wabur new --base blog Entry
+$ cd blog
+```
 
-## Try It!
+Define attributes for the data elements along with the attributes that will be
+displayed in a list view. Open `lib/ui_controller.rb` and modify by adding two
+lines and changing one.
+
+```ruby
+# encoding: UTF-8
```

Comment:
`# encoding: UTF-8` can be left out in actually.. The interpreter encodes to UTF-8 by default since Ruby 2.0